tests/test_arrowload.py

```diff
@@ -0,0 +1,330 @@
+import shutil
+
+import pytest
+
+from .conftest import TESTS_PATH, sanitize
+
+
+@pytest.fixture()
+def setup_arrow_files():
+    for filep in (TESTS_PATH / "fixtures").glob("*.arrow"):
```

Comment:
I feel like I had a reason once, but now I've no idea what it was.  Possibly I thought when I wrote the initial bash test that we didn't have access to the fixtures folder in the container, but we do. I may have just been following the gz test .do file, which loads from the output folder, but that's because it depends on a previous test that generates a file in there.